app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+def copy_file(command: str) -> None:
+    # Split the command string into its components
+    parts = command.split()
+
+    if len(parts) != 3:
+        print(
+            "Invalid command format. "
+            "Please provide command in the format: cp old_file new_file"
+        )
+        return
+
+    _, old_file, new_file = parts
+
+    # Validate if the source and destination filenames are the same
+    if old_file == new_file:
+        print(
+            "The source and destination filenames are the same. "
+            "Nothing to do."
+        )
+        return
+
+    # Validate the command
+    if parts[0] != "cp":
+        print("Invalid command. Please use 'cp' command.")
+        return
```

Comment:
You can check for len of parts, validate cp command and compare file names in one line - and then perform main logic under this if statement so you won't need returns.
Also these comments are self explanatory - no need to clutter code with something that is obvious